tests/lib/tools/store-state

```diff
@@ -72,9 +72,13 @@ make_snap_installable(){
 
     if [ -n "$snap_id" ]; then
         if ! command -v yaml2json; then
+            # FIXME: When fakestore is setup this snap cannot be installed
+            # TODO: Install remarshal in setup_fake_store
```

Comment:
Where is remarshal used? I only see it being installed but not called.

Reply:
`yaml2json` is an auto-alias to `remarshal.yaml2json`, used below to extract the snap name from `snap.yaml`.

https://github.com/canonical/snapd/blob/e440fb944764fc6340acce492450dd139ab2022b/tests/lib/tools/store-state#L84